tests/e2e-tests/src/orchestration.test.ts

```diff
@@ -25,6 +25,13 @@ describe('orchestration', () => {
       }
     });
 
+    const tokenUsage = response.getTokenUsage();
+
+    console.log(
+      `Total tokens consumed by the request: ${tokenUsage.total_tokens}\n` +
+        `Input prompt tokens consumed: ${tokenUsage.prompt_tokens}\n` +
+        `Output text completion tokens consumed: ${tokenUsage.completion_tokens}\n`
+    );
```

Comment:
This is why lint fails. This is not used anywhere, I assume this is a left over from manual testing?

Reply:
yes, removed it now.